resources/core/charts/monitoring/charts/alertmanager/README.md

```diff
@@ -20,68 +20,105 @@ metadata:
     release: {{ .Release.Name }}
   name: alertmanager-{{ .Release.Name }}
 data:
-  alertmanager.yaml: {{ toYaml .Values.config | b64enc | quote }}
+  alertmanager.yaml: |-
+    {{ include "alertmanager.yaml.tpl" . | b64enc }}
 {{- range $key, $val := .Values.templateFiles }}
   {{ $key }}: {{ $val | b64enc | quote }}
 {{- end }}
 ```
 
-The next section explains the Alertmanager configuration.
+Secret `data:` is an encoded ```alertmanager.yaml``` file which contains all the configuration for alerting notifications.
+
+The next section explains how to configure Alertmanager for enabling alerting notifications.
+
 
 #### Alertmanager configuration - alertmanager.yaml
 
-[kyma/resources/core/charts/monitoring/charts/alertmanager/values.yaml](values.yaml) pre-configure two simple receiver to handle alert in **VictorOps and Slack**.
+The template
+[kyma/resources/core/charts/monitoring/charts/alertmanager/templates/alertmanager.config.yaml](templates/secret.yaml) pre-configure two simple receiver to handle alert in **VictorOps and Slack**.
+
+This yaml file pre-configure two simple receivers to handle alert in **VictorOps and Slack**.

```

Comment:
``This yaml file pre-configures two simple receivers to handle alerts in **VictorOps and Slack**.``

Reply:
done